src/solc_0.8/polygon/child/land/PolygonLandBaseToken.sol

```diff
@@ -89,10 +123,10 @@ contract PolygonLandBaseToken is ERC721BaseToken {
         require(from != address(0), "from is zero address");
         require(to != address(0), "can't send to zero address");
         require(sizes.length == xs.length && xs.length == ys.length, "invalid data");
-        bool metaTx = msg.sender != from && isTrustedForwarder(msg.sender);
+        bool metaTx = msg.sender != from && (isTrustedForwarder(msg.sender) || hasRole(TRANSFER_ROLE, msg.sender));
         if (msg.sender != from && !metaTx) {
             require(
-                _superOperators[msg.sender] || _operatorsForAll[from][msg.sender],
+                _superOperators[msg.sender] || _operatorsForAll[from][msg.sender] || hasRole(TRANSFER_ROLE, msg.sender),
```

Comment:
Looks like this addition is redundant since metaTx will be true already in the case hasRole(TRANSFER_ROLE, msg.sender) = true